src/base/bittorrent/session.cpp

```diff
@@ -3372,8 +3372,14 @@ void Session::createTorrentHandle(const libt::torrent_handle &nativeHandle)
     bool fromMagnetUri = !torrent->hasMetadata();
 
     if (data.resumed) {
-        if (fromMagnetUri && !data.addPaused)
+        if (fromMagnetUri && data.addPaused)
```

Comment:
This is strange, and I'm not sure how I made that edit. Sorry! I've corrected it.